src/plugin-vite.js

```diff
@@ -82,6 +106,20 @@ export const Vueless = function (options = {}) {
         await cacheIcons({ mode: "vuelessIcons", env, debug, targetFiles });
         /* copy vueless cache folder */
         await copyIconsCache(mirrorCacheDir, debug);
+
+        for await (const [componentName, componentDir] of Object.entries(COMPONENTS)) {
+          const customProps =
+            componentName in vuelessConfig.component &&
+            vuelessConfig.component[componentName].props;
+
```

Comment:
please move for logic and rest related logic inside the fuctions. Now you have related logic in 2 places. This is not good.

we need to have only 2 funcs in vite plugin. for  modify and for restore other ones is internal logic which should be hidden inside of the util.